conda-forge.yml

```diff
@@ -1,6 +1,6 @@
 build_platform:
   linux_aarch64: linux_64
-  linux_ppc64le: linux_64
+  linux_ppc64le: linux_ppc64le
```

Comment:
did cross compiling really fail that hard?

Reply:
travis has been giving ALOT of problems.

Reply:
From what I see, it built fine with cross-compiling (https://github.com/conda-forge/libjxl-split-feedstock/runs/30992590715)

Reply:
it would be great if we could go back to cross compilation.

Reply:
Well just for your reference:
https://github.com/conda-forge/status/issues/185

Reply:
Excuse me for that. I didn't know Travis CI had problems since the [log](https://app.travis-ci.com/github/conda-forge/libjxl-split-feedstock/builds/272611226) looks fine and the [default in the document](https://conda-forge.org/docs/maintainer/conda_forge_yml/#build-platform) is to build natively. I will revert to cross-compilation.